salt/modules/win_service.py

```diff
@@ -827,8 +828,20 @@ def enable(name, **kwargs):
 
         salt '*' service.enable <service name>
     '''
-    modify(name, start_type='Auto')
-    return info(name)['StartType'] == 'Auto'
+    if kwargs is not None:
+        for key, value in kwargs.iteritems():
+            if key == 'StartType' and value == 'Manual':
+                modify(name, start_type='Manual')
+                return info(name)['StartType'] == 'Manual'
+            elif key == 'StartType' and value == 'Delayed':
+                modify(name, start_type='Auto', start_delayed=True)
+                return info(name)['StartTypeDelayed']
+            else:
+                log.error('Invalid argument for win_service.enable!')
+                return False
+    else:
+        modify(name, start_type='Auto')
+        return info(name)['StartType'] == 'Auto'
```

Comment:
If you add the two parameters above, you don't need the whole `if kwargs is not None:` block. Something like this should work.

```
modify(name, start_type=start_type, start_delayed=start_delayed)
status = info(name)
return statys['StartType'].lower() == start_type.lower() and status['StartTypeDelayed'] == start_delayed
```